1.9/administration/monitoring/metrics/metrics-api.md

```diff
@@ -4,29 +4,35 @@ feature_maturity: preview
 menu_order: 1
 ---
 
-The Metrics API exposes node, container, and application metrics.
+You can use the Metrics API to periodically poll for data about your cluster, hosts, containers, and applications. You can then pass this data to a third party service of your choice to achieve informative charts, dashboards, and alerts.
 
 The Metrics API is backed by the [DC/OS Metrics component](/docs/1.9/overview/architecture/components/#dcos-metrics), which runs on all nodes in the cluster.
 
-For more information about using the Metrics API, see [Metrics](/docs/1.9/administration/monitoring/metrics/).
 
 
 ## Routes
 
-Access to the Metrics API is proxied through the Admin Router on each node using the following route:
+Access to the Metrics API is proxied through the Admin Router on each node.
```

Comment:
Afaik admin router is not involved here. You connect to localhost:61001 instead.